src/components/contact/contact.js

```diff
@@ -25,8 +25,8 @@ const Contact = () => {
 };
 
 Contact.propTypes = {
-  firstNameOrNickname: PropTypes.string.isRequired,
-  fullName: PropTypes.string.isRequired,
+  firstNameOrNickname: PropTypes.string,
```

Comment:
Investigate with the other prop type problem

Reply:
these props are not being used, so they were removed